### PR TITLE
placeholder splash screen text; add Welcome screen to Account Setup

### PR DESCRIPTION
**Pull request by CliffordAndersonCambridge**

### CODE REVIEW

**Comment by MarshallMcCoy** [comment]
Pulled the branch down and looked at everything locally -- looks good. In one of my previous PRs Mithun voiced a preference for programmatic transitions between views over doing them in storyboard but my guess is that doesn't matter so much here since we aren't passing data between the views.

File: QStreak/View Controllers/Account Setup/AccountSetupViewModel.swift [hunk 1]
```diff
@@ -18,6 +18,10 @@ class AccountSetupViewModel {
 
     weak var delegate: AccountSetupViewModelDelegate?
 
+    func getStartedButtonTapped() {
```
Comment (MarshallMcCoy) [inline QStreak/View Controllers/Account Setup/AccountSetupViewModel.swift]: I don't think we need this?

Comment (CliffordAndersonCambridge) [reply QStreak/View Controllers/Account Setup/AccountSetupViewModel.swift]: Whoops, I started with something there, but forgot to take it out later. Thanks! I just pushed a revision.

**Comment by CliffordAndersonCambridge** [comment]
Good point about programmatic transitions. I'll leave it for now but let's touch on that during our next call. I'd like to gain a better sense of the trade-offs, for my own understanding.

**Review by imreddy88 — APPROVED** [review]
Add some constraints to the launch screen. Otherwise looks gtg.

**Pull request merged**